admin-tools/data_mover.sh

```diff
@@ -0,0 +1,116 @@
+#! /bin/bash
```

Comment:
Maybe this script should be explained in a .md piece of documention in some place of the repository?